lms/djangoapps/survey/tests/test_utils.py

```diff
@@ -9,7 +9,7 @@
 from django.test.client import Client
 
 from survey.models import SurveyForm
-from survey.utils import is_survey_required_and_unanswered, is_survey_required_for_course
+from survey.utils import check_survey_required_and_unanswered, is_survey_required_for_course
```

Comment:
You'll see below in survey/utils.py, we flipped the return values of `is_survey_required_and_unanswered` and made it conform to the "check" paradigm we use elsewhere.  It has very limited callers, so this was fairly simple to do.